### PR TITLE
#113 -- Added RadioGroup.Label feature.

### PR DESCRIPTION
**Pull request by nathaniellee**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

### CODE REVIEW

**Comment by nathaniellee** [comment]
David reviewed this and approved via Slack.

File: src/components/RadioGroup/examples/default.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import RadioGroup from '../RadioGroup';
+
+const style = {
+	marginRight: '13px'
```
Comment (jondlm) [inline src/components/RadioGroup/examples/default.jsx]: Is this something we should be handling in our LESS files?

Comment (ogupte) [inline src/components/RadioGroup/examples/default.jsx]: i'm not sure? should we have styles in our docs less which are for specific examples?

Comment (jondlm) [inline src/components/RadioGroup/examples/default.jsx]: My question was more, should this spacing be default for labeled components.

Comment (nathaniellee) [inline src/components/RadioGroup/examples/default.jsx]: That's a good question. These are kind of arbitrary styles. The thing of it is, though, that it's up to the consumer to decide whether to render these in a horizontal or vertical fashion... would choosing default margins put more onus on the consumer to have to modify those styles?

I can understand if it's just a tiny bit of margin all around so they're not crushed up against each other (like how different browsers have their defaults around certain controls), but I do know that for the purposes of how they show up in our application David and Ashley expressed their idea of particular spacing sizes.

**Pull request merged**